src/core/features/course/pages/list-mod-type/list-mod-type.ts

```diff
@@ -34,10 +34,14 @@ import { CoreAnalytics, CoreAnalyticsEventType } from '@services/analytics';
 })
 export class CoreCourseListModTypePage implements OnInit {
 
+    static readonly LOAD_MORE_ACTIVITIES = 20; // How many activities should load each time showMoreActivities is called.
```

Comment:
```suggestion
    private static readonly PAGE_LENGTH = 20; // How many activities should load each time showMoreActivities is called.
```

Besides the naming, this could be a constant declared in the function where it is used, given that it's only used in one place at the moment.